prov/efa/src/rdm/efa_rdm_ep_fiops.c

```diff
@@ -844,9 +844,9 @@ void efa_rdm_ep_wait_send(struct efa_rdm_ep *efa_rdm_ep)
 	while (efa_rdm_ep_has_unfinished_send(efa_rdm_ep)) {
 		/* poll cq until empty */
 		if (tx_cq)
-			efa_rdm_cq_poll_ibv_cq(-1, &tx_cq->ibv_cq);
+			efa_rdm_cq_poll_ibv_cq_closing_ep(&tx_cq->ibv_cq, efa_rdm_ep);
 		if (rx_cq)
-			efa_rdm_cq_poll_ibv_cq(-1, &rx_cq->ibv_cq);
+			efa_rdm_cq_poll_ibv_cq_closing_ep(&rx_cq->ibv_cq, efa_rdm_ep);
```

Comment:
are you still going to call `efa_domain_progress_rdm_peers_and_queues` which can repost the earlier queued packets for the closing ep?

Reply:
If we still need to post the RNRed EOR/RECEIPT pkts earlier, that will be a pain to have another `closing_ep` version of `efa_domain_progress_rdm_peers_and_queues`